extensions/mqtt/processors/ConsumeMQTT.h

```diff
@@ -125,14 +119,14 @@ class ConsumeMQTT : public processors::AbstractMQTTProcessor {
   void onSubscriptionSuccess();
   void onSubscriptionFailure(MQTTAsync_failureData* response);
   void onSubscriptionFailure5(MQTTAsync_failureData5* response);
-  void onMessageReceived(char* topic_name, int /*topic_len*/, MQTTAsync_message* message) override;
+  void onMessageReceived(std::string topic, std::unique_ptr<MQTTAsync_message, MQTTMessageDeleter> message) override;
```

Comment:
this seems to be `SmartMessage`

Reply:
Moved struct definition to `AbstractMQTTProcessor`.